### ISSUE TITLE
Override commentsRemove callback and call parent

### ISSUE THREAD

**Issue by raulmellado** [handsontable/handsontable#2532]
Hi everyone!

Not sure if this is a bug or a feature, but when I delete a comment (using 0.15.0b5), the afterSetCellMeta event is not called (if I add a comment, it gets invoked).

So what I tried is editing the commentsRemove callback, but then the comment does not get removed :-(

Maybe this is more of a js question (js is not my speciality), but how can I have my own callback but still call the previous functionality?

This is my code:

```
    config.contextMenu = {
    items: {
             "commentsRemove":{
                         name: 'Remove Comment',
             callback: (function(key, selection) {
                              // my own code here
                              // how to call the previous code to actually remove the comment?
                         });
```

Thanks a lot for your help!!

**Comment by AMBudnik** [comment]
`afterSetCellMeta` is fired after adding comments and editing it but not after deleting the comment. Thank you @raulmellado for that suggestion.

**Comment by rjansen72** [comment]
I added a afterReniveCellMeta in our fork for just this purpose. We persist the comments in our backend, and thus need to remove them based on user action. @raulmellado let me know if you want to take a look at what we did to implement this.

**Comment by ilya4544** [comment]
Hi!
`afterSetCellMeta` is still not fired after deleting the comment.

How to catch this event?

**Comment by AMBudnik** [comment]
related: https://github.com/handsontable/handsontable/issues/3899

**Comment by wszymanski** [comment]
Fixed within the task #3899.
